src/dist/cmd/command_types.cpp

```diff
@@ -149,6 +149,6 @@ void command::printTo(std::ostream &out) const
         << "arguments=" << to_string(arguments);
     out << ")";
 }
-} // namespace cmd
-} // namespace dist
-} // namespace dsn
+}
+}
```

Comment:
Why these auto generated code has been removed?

Reply:
It seems that code auto generated by thrift won't add namespace comments at the end of the file. You can see other files generated by thrift, such as `include/dsn/dist/replication/replication_types.h`